start.markdown

```diff
@@ -3,7 +3,64 @@ title: Getting Started
 layout: main
 ---
 
-- [Getting O2](/git)
-- [Building O2](/building)
-- [Running O2](/running)
-- [Further material](/links)
+Get O²
+======
+
+The ALICE O² Software is hosted on [GitHub](https://github.com) under the
+[AliceO2Group](https://github.com/AliceO2Group/) organization. The core package is
+[AliceO2](https://github.com/AliceO2Group/AliceO2).
+
+
+Build O²
+========
+
+In order to build the ALICE O² software we use [aliBuild](https://alisw.github.io/alibuild). Check
+out our [build instructions](https://alice-doc.github.io/alice-analysis-tutorial/building/).
+
+
+Run O²
+======
+
+The simplest way to run `AliceO2` is by using directly our nightly builds from CVMFS. If you have a
+CERN account, you can connect via SSH to `lxplus7.cern.ch` first to have CVMFS available.
+
+This is how you list all available builds:
+
+    /cvmfs/alice-nightlies.cern.ch/bin/alienv q | grep O2::nightly
+
+And this is how you load one of them:
+
+    /cvmfs/alice-nightlies.cern.ch/bin/alienv enter VO_ALICE@O2::nightly-20171130-1
+
+Nightly builds are also available on selected Grid sites (the ones supporting the
+`alice-nightlies.cern.ch` CVMFS repository), allowing for off-CERN testing.
+
+Nightly builds are automatically announced on the <alice-o2-software-support@cern.ch> mailing list.
+Read on the [Get support](#get-support) section to know how to subscribe.
+
+
+Contribute to O²
+================
+
+Since O² uses the Git version control system it is recommended you follow [our AliPhysics Git
+tutorial](http://alisw.github.io/git-tutorial/) to learn its rudiments.
+
+
+Get support
+===========
+
+For simple support questions (_e.g. I cannot build O2..._) you can use the
+<alice-o2-software-support@cern.ch> mailing list.
+
+> [Click here to subscribe to alice-o2-software-support@cern.ch](https://e-groups.cern.ch/e-groups/EgroupsSubscription.do?egroupName=alice-o2-software-support)
+
+**Note:** you need to be a registered ALICE member in order to do that.
+
+You might want to [have a look at the mailing list archive](https://groups.cern.ch/group/alice-o2-software-support)
+to avoid posting duplicate questions.
+
+If you have problems subscribing, check with the [ALICE Secretariat](mailto:alice.secretariat@cern.ch)
+if you are correctly registered as ALICE member (just send them an email).
+
+If you think you have found a bug in our software, instead of reporting it on the mailing list you
+are encouraged to use [JIRA](https://alice.its.cern.ch).
```

Comment:
can we make the link such that it directly opens a new ticket for the O2 project flagged as bug?

Reply:
I tried but I failed... in any case, the page refers to O2 in general, not specifically to the AliceO2 project, and we have many JIRA projects. We might, instead (but I'd do it as a next step, not here) have a detailed description on how to choose the JIRA project where to open the bug.

Reply:
Oh wait I have found [this](https://confluence.atlassian.com/jirakb/creating-issues-via-direct-html-links-159474.html)! Let me try...

Reply:
No way, it does not work any longer with the new JIRA. Oh well.

Reply:
For the records, we just discussed with Dario and here are ways of doing it : 
[https://alice.its.cern.ch/jira/secure/CreateIssue.jspa?pid=11201&issuetype=1](https://alice.its.cern.ch/jira/secure/CreateIssue.jspa?pid=11201&issuetype=1)  opens a new bug in OCCF.
[https://alice.its.cern.ch/jira/secure/CreateIssue.jspa?pid=11201](https://alice.its.cern.ch/jira/secure/CreateIssue.jspa?pid=11201) lets the user modify project and issue type

Reply:
OK, found the solution, now implemented, thanks to @Barthelemy. Now we have a direct link!